### PR TITLE
Scalatest version upgrade (BW-933).

### PR DESCRIPTION
**Pull request by cahrens**
Original Scala-Steward PR: https://github.com/broadinstitute/cromwell/pull/6532

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -40,7 +40,7 @@ object Dependencies {
   private val ficusV = "1.5.1"
   // The "com.vladsch.flexmark" % "flexmark-profile-pegdown" % flexmarkV dependency is an implicit, version-specific
   // runtime dependency of ScalaTest. At the time of this writing this is the newest version known to work.
```
Comment (mcovarr) [inline project/Dependencies.scala:42]: I wish I remember what "known to work" meant, that's a pretty big jump in the version of this dependency for a patch release bump in ScalaTest. Anyhoo I think this comment may be obsolete now.

File: project/Dependencies.scala [hunk 2]
```diff
@@ -40,7 +40,7 @@ object Dependencies {
...
-  private val flexmarkV = "0.36.8" // scala-steward:off
+  private val flexmarkV = "0.62.2" // scala-steward:off
```
Comment (mcovarr) [inline project/Dependencies.scala:43]: can the `scala-steward:off` go away now?

File: project/Dependencies.scala (hunk 1 above)
Comment (cahrens) [reply project/Dependencies.scala:42]: Scalatest requires flexmark, but for some reason doesn't bring it in itself (I double-checked this removing flexmark from Dependencies, which bombed). But the version of flexmark needs to match what Scalatest expects. For the release of Scalatest that we are upgrading to, they want version 0.62.2 (see notes in https://github.com/scalatest/scalatest/releases/tag/release-3.2.10). Upgrading Scalatest without upgrading flexmark failed.

I think the comment is accurate, though a bit obtuse.

File: project/Dependencies.scala (hunk 2 above)
Comment (cahrens) [reply project/Dependencies.scala:43]: No, I don't think that's a good idea. If we try to upgrade flexmark to a version beyond what Scalatest supports, we could see errors. Scalatest and flexmark should be upgraded together, which I tried to explain with my inline comment.

File: project/Dependencies.scala (hunk 1 above)
Comment (mcovarr) [reply project/Dependencies.scala:42]: Perhaps add some of this detail to the `scalatestV` comment below? It's kind of a weird situation that Scalatest doesn't express its actual dependency on flexmark...

**Review by mcovarr — APPROVED** [review]

**Comment by cahrens** [comment]
@mcovarr I tried to make the comments more explicit. Let me know what you think.

**Review by cjllanwarne — APPROVED** [review]

**Review by mcovarr — APPROVED** [review]
LGTM, thanks!

**Pull request merged**